### ISSUE TITLE
tax_total in line_items missing? 

### ISSUE THREAD

**Issue by chewlim** [marketplacer/api.marketplacer.com#29]
Hi there, 

I have noticed that the field attributes->tax_total within the line_items missing from the "GET /api/v2/client/invoices/1"? 

Is this bug or new changes? 

```
 {
      "id": 1,
      "type": "line_items",
      "attributes": {
        "price": "100.0",
        "status": "allocated",
        "subtotal": "100.",
        "tax_total": "0.0",
        "total": "100.0",
        "quantity": 1,
        "taxon_name": "Hybrid Bikes",
        "brand_name": "Giant",
        "advert_name": "Giant Cypress DX",
        "advert_external_id": "ABC123",
        "variant_name": "Navy Blue / Medium",
        "variant_sku": null,
        "variant_barcode": "08717611735314",
        "variant_external_id": "DEF123"
      }
    }

```

**Comment by michaeldawson** [comment]
Hi chewlim, apologies - this field shouldn't have been removed. 

We're introducing some new features that require that tax be calculated at the invoice level, and not at the line item level. This field shouldn't have been removed from this API endpoint response, but rather just set to $0.

The reason for this is because new invoice-level discounts can lower the amount of tax payable on the invoice, without necessarily applying to a particular line item.

I've restored this field to the API response - apologies again, and let me know if you have more issues, or if you'd like more information on this.
